justfile

```diff
@@ -59,7 +59,13 @@ install-denv:
 
 # configure how ldmx-sw will be built
 configure *CONFIG:
-    denv cmake -B build -S . {{ CONFIG }}
+    denv cmake -B build -S .  -DADDITIONAL_WARNINGS=ON  -DENABLE_CLANG_TIDY=ON  {{ CONFIG }}
```

Comment:
If we really want these CMake options to be ON by default, maybe we should put them into the CMakeLists.txt like that?

e.g. I think the following would still allow us to disable on the command line but would change the default to `ON` if not specified.

```cmake
if (NOT ADDITIONAL_WARNINGS)
  set(ADDITIONAL_WARNINGS ON)
endif()
```

Reply:
So it's yes and no. It does seem to work, but gives this other warning
```
CMake Warning (dev) at cmake/CompilerWarnings.cmake:13 (option):
  Policy CMP0077 is not set: option() honors normal variables.  Run "cmake
  --help-policy CMP0077" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  For compatibility with older versions of CMake, option is clearing the
  normal variable 'ADDITIONAL_WARNINGS'.
Call Stack (most recent call first):
  cmake/BuildMacros.cmake:143 (enable_compiler_warnings)
  Recon/CMakeLists.txt:37 (setup_library)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at cmake/StaticAnalysis.cmake:5 (option):
  Policy CMP0077 is not set: option() honors normal variables.  Run "cmake
  --help-policy CMP0077" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  For compatibility with older versions of CMake, option is clearing the
  normal variable 'ENABLE_CLANG_TIDY'.
Call Stack (most recent call first):
  cmake/BuildMacros.cmake:145 (enable_clang_tidy)
  Recon/CMakeLists.txt:37 (setup_library)
This warning is for project developers.  Use -Wno-dev to suppress it.
```
which makes it pretty ugly... so I'll just keep this as it was. Honestly I think it's more transparent too to have it explicitly like this in the justfile